src/EnergyPlus/DataSurfaces.cc

```diff
@@ -86,6 +94,14 @@ namespace DataSurfaces {
 	using namespace DataPrecisionGlobals;
 	using namespace DataVectorTypes;
 	using namespace DataBSDFWindow;
+  using namespace DataHeatBalance;
```

Comment:
You have numerous indention issues throughout the code. Looks like most are spaces when they should be tabs